pkg/policy/consumer.go

```diff
@@ -86,11 +89,13 @@ type Consumable struct {
 	// key (security identity) should be garbage collected upon policy calculation.
 	EgressIdentities map[identity.NumericIdentity]bool `json:"egress-identities"`
 
-	// L4Policy contains the policy of this consumable
+	// L4Policy contains the L4-only policy of this consumable
```

Comment:
This also includes Label-dependent L4 policies, doesn't it?

Reply:
I'll double check and confirm this.

Reply:
You're right. The naming is not clear at all :(